attributes/default.rb

```diff
@@ -20,7 +20,7 @@
 ]
 
 default['snipeit']['database']['domain'] = 'corp.yourcompany.com'
-default['snipeit']['database']['hostname'] = node['hostname']
+default['snipeit']['database']['hostname'] = '127.0.0.1'
```

Comment:
In the past, I've seen that there are differences between using `localhost` and '127.0.0.1` - just curious if you have seen those and why you might've chosen one over the other for the default.

Reply:
In Laravel, I'd need to specify the Unix socket for `DB_SOCKET` in order to specify `localhost` as the hostname. Snipe-IT supports this. However, this just sacrifices performance over simplicity.